terraform/account/archive.tf

```diff
@@ -1,4 +1,5 @@
 #KMS key for Server Side Encryption (SSE) of S3 buckets
+#tfsec:ignore:AWS019
```

Comment:
What's this for?

Reply:
this is a tfsec override,we're trialling this. this override is to ignore the rule for bucket logging. no-one other than ops will have access). see https://github.com/tfsec/tfsec for details of the tool.